services/brig/src/Brig/API.hs

```diff
@@ -813,12 +813,9 @@ sitemap o = do
   post "/onboarding/v3" (continue onboardingH) $
     accept "application" "json"
       .&. header "Z-User"
-      .&. jsonRequest @AddressBook
+      .&. jsonRequest @Value
   document "POST" "onboardingV3" $ do
-    Doc.summary "Upload contacts and invoke matching. Returns the list of Matches"
-    Doc.body (Doc.ref Doc.addressBook) $ Doc.description "Address book"
-    Doc.returns (Doc.ref Doc.onboardingMatches)
-    Doc.response 200 "Matches" Doc.end
+    Doc.summary "[DEPRECATED]  the end-point does nothing.  (it used to upload contacts and invoke matching.)"
```

Comment:
You can even mark the endpoint as deprecated in Swagger, look at how we do it for `deprecatedCompletePasswordResetH`.

Reply:
(you can also move the endpoint to the "Deprecated" section)